controllers/monitoring_controller.go

```diff
@@ -241,12 +242,12 @@ func (r *MonitoringReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, err
 	}
 
-	log.Info("Monitoring Controller reconciling.")
-	err = r.reconcileRoleBinding(ctx, req)
-	if err != nil {
-		return ctrl.Result{}, err
-	}
-	log.Info("Monitoring Controller reconciled successfully.")
+	// log.Info("Monitoring Controller reconciling.")
```

Comment:
Complete `monitoring_controller.go` file is resposible for doing `reconcileRoleBinding`. If these reconcilation is not needed anymore then I would suggest to remove this complete controller and all its references.